FSAE.X/FSAE_config.c

```diff
@@ -325,14 +325,16 @@ void init_peripheral_modules(void) {
   SPI4CONbits.ON = 0;
   PMD5bits.SPI4MD = 1;
 
+#ifndef ECM064
```

Comment:
Yes, it's the original chip that I wrote the 32bit library for. Smaller and cheaper, convenient for testing stuff. I made the crank/cam simulation board with it.  Only a few small additions to the library files so I thought it would be useful to save